templates/tracker.mustache

```diff
@@ -148,18 +148,37 @@
                     tbody.children[i+1].appendChild(emptyNode);
                     break;
                 }
-                textnode = document.createTextNode(attempts[attempt_id]&&"Proctor link"||""); 
-
+                textnode = document.createTextNode(attempts[attempt_id]&&"Proctor link"||"");
                 node = document.createElement("a");    
                 node.setAttribute("title", "Proview Link");
                 node.setAttribute("href", attempts[attempt_id]&&attempts[attempt_id].proview_url||"");
                 node.setAttribute("target", "proview");
-                node.addEventListener("click",()=>{                                             {{! Event to display I frame containing Proview admin }}
-                    table.setAttribute("style","display: none");
-                    modal.setAttribute("style","display: block");
+                node.addEventListener("click",()=>{
+                const proctor_type = attempts[attempt_id].proctor_type;
+                     if(proctor_type=='ai_proctor'||proctor_type=='record_and_review'||proctor_type=='live_proctor'){
+                            auth_token =`{{auth_token}}`;
+                            let external_session_id='';
+                            const quiz_id=attempts[attempt_id].quiz_id;
+                            const user_id=attempts[attempt_id].user_id;
+                            const attempt_no=attempts[attempt_id].attempt_no;
+                            const external_attendee_id=user_id;
+                            if(proctor_type=='ai_proctor'||proctor_type=='record_and_review'){
+                                 external_session_id= quiz_id+'-'+user_id+'-'+attempt_no;
+                             }
+                            else if(proctor_type=='live_proctor'){
+                                     external_session_id= quiz_id+'-'+user_id;
+                            }
+                            const proctor_token= '{{token}}';
```

Comment:
@rohansharmasitoula Check the estimates for remaining this to proctor_token instead of token, code is ambiguous as there are multiple tokens being used